core/src/test/scala/unit/kafka/admin/ConsumerGroupCommandTest.scala

```diff
@@ -116,6 +123,18 @@ class ConsumerGroupCommandTest extends KafkaServerTestHarness {
 }
 
 object ConsumerGroupCommandTest {
+  // We want to test the following combinations:
+  // * ZooKeeper and the classic group protocol.
+  // * KRaft and the classic group protocol.
+  // * KRaft with the new group coordinator enabled and the classic group protocol.
+  // * KRaft with the new group coordinator enabled and the consumer group protocol.
+  def getTestQuorumAndGroupProtocolParametersAll: java.util.stream.Stream[Arguments] = {
```

Comment:
We already have the same one defined in BaseConsumerTest. I would if we could reuse it. Could we?

Reply:
I wasn't able to re use it so I added the code here but I can double check